proxy/coco/src/request.rs

```diff
@@ -71,6 +71,53 @@ pub struct Request<S, T> {
     state: S,
 }
 
+/// Wrapper for `std::Instant` to serialize using `serde_millis`.
+struct WithMillis(Instant);
+
+impl Serialize for WithMillis {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        let WithMillis(ts) = self;
+        serde_millis::serialize(ts, serializer)
+    }
+}
+
+impl<T> Serialize for Request<T, Instant>
+where
+    T: Serialize,
+{
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        let mut state = serializer.serialize_struct("Request", 4)?;
+        state.serialize_field("urn", &self.urn)?;
+        state.serialize_field("attempts", &self.attempts)?;
+        state.serialize_field("timestamp", &WithMillis(self.timestamp))?;
```

Comment:
@sarahscott I think we could just do `serde_millis::serialize(self.timestamp)` here and remove  `WithMillis`